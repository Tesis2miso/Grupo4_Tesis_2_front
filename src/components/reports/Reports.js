import Toolbar from '@mui/material/Toolbar';

function Reports(props) {
  return (
    <div>
      <Toolbar />
      Reports
    </div>
  );
}

export default Reports;
