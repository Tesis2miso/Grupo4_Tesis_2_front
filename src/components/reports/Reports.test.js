import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import Reports from './Reports';

const mockedUsedNavigate = jest.fn();
jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'),
  useNavigate: () => mockedUsedNavigate,
}));
jest.mock('axios', () => jest.fn());

describe('Tests_Reports', () => {
  let wrapper = null;

  beforeEach(() => {
    wrapper = shallow(<Reports loggedIn={true} setToken={() => { }} />);
  });

  test('show reports', () => {
    expect(wrapper).toMatchSnapshot();
  })
})