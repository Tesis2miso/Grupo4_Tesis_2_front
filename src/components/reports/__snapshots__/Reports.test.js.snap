// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests_Reports show reports 1`] = `
<div>
  <ForwardRef(Toolbar) />
  Reports
</div>
`;
